docs/Apache-HTTP-Server-Logs-Collection.md

```diff
@@ -4,13 +4,13 @@
 
 2. Download and install [OMS Agent for Linux](https://github.com/Microsoft/OMS-Agent-for-Linux) on the machine.
 
```

Comment:
Add a step here about acquiring the workspace ID for the workspace that they want the logs to be sent to.